### PR TITLE
added attachment entity and liquidbase skript

### PR DESCRIPTION
**Pull request by Bohadan-Markatov**

### CODE REVIEW

File: src/main/resources/db/changelog/changes/02-create-attachment-table.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                  constraints:
+                    nullable: false
+              - column:
+                  name: file_name
+                  type: varchar(100)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: upload_date
+                  type: datetime
```
Comment (VlasenkoYevhenii) [inline src/main/resources/db/changelog/changes/02-create-attachment-table.yaml]: type: time stamp?

File: src/main/java/com/example/taskmanagementapplication/model/Attachment.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
+    @Column(nullable = false)
+    private String dropboxFileId;
+    @Column(nullable = false)
+    private String fileName;
+    @Column(nullable = false)
+    private LocalDateTime uploadDate;
+    @Column(nullable = false)
+    private boolean isDeleted;
```
Comment (VlasenkoYevhenii) [inline src/main/java/com/example/taskmanagementapplication/model/Attachment.java:44]: use @SoftDelete annotation to get rid of the excess field in the model
image

**Review by VlasenkoYevhenii — CHANGES_REQUESTED** [review]

**Review by bodya4243 — APPROVED** [review]
all good

File: src/main/resources/db/changelog/changes/05-create-attachment-table.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
```
Comment (VlasenkoYevhenii) [inline src/main/resources/db/changelog/changes/05-create-attachment-table.yaml:2]: delete liquibase related things

**Review by VlasenkoYevhenii — CHANGES_REQUESTED** [review]

**Pull request closed without merging**